prowler/providers/aws/exceptions/exceptions.py

```diff
@@ -8,188 +8,166 @@ class AWSBaseException(ProwlerException):
-            "file": "{file}",
-            "provider": "AWS",
         },
         (1910, "AWSIAMRoleARNServiceNotIAMnorSTS"): {
             "message": "AWS IAM Role ARN service is not IAM nor STS",
             "remediation": "Check the AWS IAM Role ARN service and ensure it is either IAM or STS, visit https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html#identifiers-arns for more information.",
-            "file": "{file}",
-            "provider": "AWS",
         },
         (1911, "AWSIAMRoleARNInvalidAccountID"): {
             "message": "AWS IAM Role ARN account ID is invalid",
             "remediation": "Check the AWS IAM Role ARN account ID and ensure it is a valid 12-digit number, visit https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html#identifiers-arns for more information.",
-            "file": "{file}",
-            "provider": "AWS",
         },
         (1912, "AWSIAMRoleARNInvalidResourceType"): {
             "message": "AWS IAM Role ARN resource type is invalid",
             "remediation": "Check the AWS IAM Role ARN resource type and ensure it is valid, resources types are: role, user, assumed-role, root, federated-user, visit https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html#identifiers-arns for more information.",
-            "file": "{file}",
-            "provider": "AWS",
         },
         (1913, "AWSIAMRoleARNEmptyResource"): {
             "message": "AWS IAM Role ARN resource is empty",
             "remediation": "Check the AWS IAM Role ARN resource and ensure it is not empty, visit https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html#identifiers-arns for more information.",
-            "file": "{file}",
-            "provider": "AWS",
         },
         (1914, "AWSAssumeRoleError"): {
             "message": "AWS assume role error",
-            "remediation": "Check the AWS assume role configuration and ensure it is properly set up, please visithttps://docs.prowler.com/projects/prowler-open-source/en/latest/tutorials/aws/role-assumption/ and https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_terms-and-concepts.html#iam-term-role-session-name",
-            "file": "{file}",
-            "provider": "AWS",
+            "remediation": "Check the AWS assume role configuration and ensure it is properly set up, please visit https://docs.prowler.com/projects/prowler-open-source/en/latest/tutorials/aws/role-assumption/ and https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_terms-and-concepts.html#iam-term-role-session-name",
         },
     }
 
-    def __init__(self, code, provider="AWS", file=None, original_exception=None):
+    def __init__(
+        self, code, provider="AWS", file=None, original_exception=None, message=None
```

Comment:
Do you want to allow override the `provider`?

Reply:
In that case we can remove the argument and set it within.

Reply:
Done